index.html

```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Personal Portfolio</title>
+    <link rel="stylesheet" href="css/style.css">
+</head>
+<body>
+    <nav class ="main-nav">
+        <div class="logo">
+        <a href="#"><p>Worash Aboche</p></a>  
+        </div>
```

Comment:
- [ ] Please use the same font family for the logo as in Figma design 
![image](https://user-images.githubusercontent.com/74744787/164340296-20e696b3-95d3-4876-b6bf-908d8643f6e0.png)